sklearn/metrics/_plot/roc_curve.py

```diff
@@ -165,8 +165,9 @@ def plot_roc_curve(estimator, X, y, sample_weight=None,
     """
     check_matplotlib_support('plot_roc_curve')
 
-    classification_error = ("{} should be a binary classifer".format(
-        estimator.__class__.__name__))
+    classification_error = (
+        f"{estimator.__class__.__name__} should be a binary classifer"
```

Comment:
Do we want to backport this fix?

Reply:
We could. I am seeing the same bug in `plot_precision_recall_curve` PR to come :)

Reply:
ah I should complete my thought. I meant to say: "If we want to backport the fix, we can not use f-strings for now."

Reply:
Oh I see good point. I will revert.